core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -1033,7 +1034,9 @@ class GroupCoordinator(val brokerId: Int,
     group.transitionTo(PreparingRebalance)
 
     info(s"Preparing to rebalance group ${group.groupId} in state ${group.currentState} with old generation " +
-      s"${group.generationId} (${Topic.GROUP_METADATA_TOPIC_NAME}-${partitionFor(group.groupId)}) (reason: $reason)")
+      s"${group.generationId} (${Topic.GROUP_METADATA_TOPIC_NAME}-${partitionFor(group.groupId)}) (reason: $reason), " +
+      s"the remaining instances are ${group.allMembers}, and the delayed rebalance is $startEmpty " +
```

Comment:
This line is a bit confusing: the delayed rebalance is true / false, what does this mean?